src/init/features/project.ts

```diff
@@ -0,0 +1,133 @@
+import * as clc from "cli-color";
+import * as _ from "lodash";
+
+import { FirebaseProject, getProject, listProjects } from "../../firebaseApi";
+import * as logger from "../../logger";
+import { promptOnce, Question } from "../../prompt";
+import * as utils from "../../utils";
+
+const NO_PROJECT = "[don't setup a default project]";
+const NEW_PROJECT = "[create a new project]";
+
+/**
+ * Used in init flows to keep information about the project - basically
+ * a shorter version of FirebaseProject with some additional fields.
+ */
+export interface ProjectInfo {
```

Comment:
After reading the docs below, I think the `instance`, `location`, and `label` should all have `?` after them? I think that's also why you're being required to `as ProjectInfo` your value in the return statement of `getProjectInfo`.

Reply:
Done.